src/test/test_circuitbuild.c

```diff
@@ -219,6 +219,7 @@ test_circuit_extend_state_valid(void *arg)
   expect_log_msg("Got an extend cell, but running as a client. Closing.\n");
   mock_clean_saved_logs();
 
+#ifndef ALL_BUGS_ARE_FATAL
```

Comment:
In cases where these blocks cover an entire test, the test should use TT_SKIP() or tt_skip() rather than ifdef.  But that can be another ticket.

Reply:
I just checked all instances of ALL_BUGS_ARE_FATAL, and I couldn't find any which:
* cover the entire unit test, and
* #ifdef out the code.
That code wouldn't compile anyway, because "done" would be unused.

For future reference, if we do find any of these blocks, they need a coverity / clang analyser wrapper, like this:
https://github.com/torproject/tor/blob/3253c357eeae3434da62bf720a451aa19f0ddd32/src/test/test_circuitbuild.c#L130